common/survey/nodeDef.js

```diff
@@ -42,6 +42,8 @@ const getNodeDefLabel = (nodeDef, lang) => R.pipe(
   R.prop(lang),
 )(nodeDef)
 
+const getNodeDefValidation = (nodeDef, field) => R.path(['validation', 'fields', field])(nodeDef)
```

Comment:
getNodeDefValidation would be a method that returns all validation object associated to the nodeDef. 
```javascript
const getNodeDefValidation = getProp('validation')
```
Then you might have another function to access filed specific validation, like 
```javascript
 const getNodeDefFieldValidation = (nodeDef, field) => R.pipe(
getNodeDefValidation,
R.path(['fields', field])
)(nodeDef)
```

Reply:
Good suggestion!